src/Service/Controllers/RestController.cs

```diff
@@ -241,11 +293,28 @@ private async Task<IActionResult> HandleOperation(
                     HttpContextExtensions.GetLoggerCorrelationId(HttpContext));
 
                 Response.StatusCode = (int)HttpStatusCode.InternalServerError;
+                if(activity is not null)
+                {
+                    activity.TrackRestControllerActivityFinishedWithWithException(ex, Response.StatusCode);
+                }
```

Comment:
Can't this be simplified as:

```suggestion
                activity?.TrackRestControllerActivityFinishedWithWithException(ex, Response.StatusCode);
```